test/helpers/elasticsearch.go

```diff
@@ -132,6 +132,10 @@ func (es *ElasticLogStore) HasAuditLogs(timeToWait time.Duration) (bool, error)
 	return true, err
 }
 
+func (es *ElasticLogStore) ClusterLocalEndpoint() string {
```

Comment:
What does this mean "ClusterLocalEndpoint"

Reply:
It means you provide a service object for your store and this method gives you the cluster local DNS name e.g. `kafka.openshift-logging.svc.cluster.local`